### ISSUE TITLE
Bogue saving in the cms

### ISSUE THREAD

**Issue by MaudeFJ** [culturecreates/tech-support-tickets#167]
https://github.com/user-attachments/assets/92ace5a8-44c8-402a-aac9-9461056ddfae

**Comment by MaudeFJ** [comment]
problem In all 4 of those event https://cms.footlight.io/dashboard/63457978637093005071a053/events?page=1&order=asc&sortBy=upcomingEventDate&query=belles+heures

**Comment by troughc** [comment]
@AbhishekPAnil I confirmed that this is happening for me as well.

**Comment by AbhishekPAnil** [comment]
Bug Report: Accessibility Note Deletion Not Working

Issue: When a user removes the last remaining language literal from an existing event's accessibility note field, the PATCH update sends undefined (omits the field) instead of null. The backend cannot distinguish between "no change" and "user deleted the value", causing the accessibility note to persist in the database despite user removal. 

Fix: Modified the code to explicitly send accessibilityNote: null when the field is cleared but previously had a value, allowing the backend to properly delete the field

**Comment by troughc** [comment]
@AbhishekPAnil where do I test this? Thanks.

**Comment by AbhishekPAnil** [comment]
> [@AbhishekPAnil](https://github.com/AbhishekPAnil) where do I test this? Thanks.


You can use this [PR link](https://pr-2077.dla06mbe0wyl7.amplifyapp.com/)
Since its a tech-support issue we wont able to link it to a PR.

cc @troughc

**Comment by troughc** [comment]
Tested: PASSED 
Please deploy in the next window.

**Comment by troughc** [comment]
@MaudeFJ this is now fixed.
